lib/facts/linux/hypervisors/docker.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Facts
+  module Linux
+    module Hypervisors
+      class Docker
+        FACT_NAME = 'hypervisors.docker'
+
+        def call_the_resolver
+          fact_value = check_docker
+          Facter::ResolvedFact.new(FACT_NAME, fact_value)
+        end
+
+        def check_docker
+          info = Facter::Resolvers::DockerLxc.resolve(:hypervisor)
+          info.values.first if info && info.keys.first == :docker
```

Comment:
I think if you write it like this  
`info[:docker] if info`
It would be a bit easier to understand

Reply:
Done